src/components/Form/DebloatStub.js

```diff
@@ -0,0 +1,46 @@
+import validatePassword from '../../validators/validatePassword'
+import EmailFormatValidator from '../../validators/EmailValidator'
+
+// This might be plucked out of routeData.data.emailpage.fields or something similar
+const json = {
+  email: {
+    placeholder: 'example@ethoslife.com',
+    name: 'the-email',
+    labelCopy: 'Email ',
+    tid: 'the-email-tid',
+    validators: ['EmailFormatValidator'],
```

Comment:
turns out to be easy to map these one to one if we simply use a lookup table. Also, if we change the names we just do:

```js
const validatorsTable = {
  validatePassword: IDecidedToMakeItWeirdAndRenameValidatePasswordButIStillWork
...
```